content/de/docs/glossary.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Glossar
+slug: glossary
+top_graphic: 1
+date: 2018-12-30
+---
+
+{{< lastmod >}}
+
+<!--
+Note for translators:
+ 
+- Usage of the "def" macro (in other languages than English):
+{% def 
+	id="a unique id for anchor - the SAME than for english. will be prefixed by `def-`" 
+	name="The term to define (optional if english or abbr is provided)"
+	abbr="an accronym (optional)"
+	english="the english term (optional - if present the abbr is in english too)" %}}
+		the definition
+{% /def %}
+	
+- Check the javascript console for errors.
+
+- Automatic titles on definition's link cuts everything after the last point (to remove source links)
+
+-->
+
+{{% def id="AIA" name="Authority Information Access" abbr="AIA" %}} Eine Zertifikat [Erweiterung](#def-extension) benutzt zum Identifizieren von [User Agents](#def-user-agent), wie sie Informationen über Aussteller des [Zertifikat](#def-certificate) identifizieren. Typischerweise spezifiziert sie die [OCSP](#def-OCSP) URI und die [issuer URI](#def-CAI). {{% /def %}}
+
+{{% def id="ACME" name="Automatic Certificate Management Environment" abbr="ACME" abbr_first="1" %}} Das Protokoll implementiert von [Let's Encrypt](#def-LE). Software, die kompatibel zu diesem Protokoll ist, kann zur Kommunikation mit Let's Encrypt benutzt werden, um nach einem [Zertifikat](#def-leaf) zu fragen. [ACME Entwurf](https://tools.ietf.org/html/draft-ietf-acme-acme-16) - [Wikipedia](https://en.wikipedia.org/wiki/Automated_Certificate_Management_Environment) {{% /def %}}
```

Comment:
```suggestion
{{% def id="ACME" name="Automatic Certificate Management Environment" abbr="ACME" abbr_first="1" %}} Das Protokoll implementiert von [Let's Encrypt](#def-LE). Software, die kompatibel zu diesem Protokoll ist, kann zur Kommunikation mit Let's Encrypt benutzt werden, um nach einem [Zertifikat](#def-leaf) zu fragen. [ACME Entwurf](https://tools.ietf.org/html/draft-ietf-acme-acme-16) - [Wikipedia](https://de.wikipedia.org/wiki/Automatic_Certificate_Management_Environment) {{% /def %}}
```
Deutsch Wikipedia

Reply:
I checked several Wikipedia links in German. The explanation are often not so good as in English. I would leave here the original link here and the user can decide on Wikipedia to use the German version. Otherwise it's too confusing sometimes to use the German version and sometimes not.